README.md

```diff
@@ -3,14 +3,48 @@
 This is a library to help with authenticating requests to and from PeekPro when
 writing apps.
 
-## Config:
+## Configuration
+
+### Basic Configuration
 
 ```elixir
 config :peek_app_sdk,
   peek_app_secret: "APP_SECRET",
-  peek_app_id: "APP_ID"
+  peek_app_id: "APP_ID",
+  peek_api_url: "https://api.peek.com",
+  peek_app_key: "APP_KEY"
+```
+
+### Multiple Application Configurations
+
+PeekAppSDK supports multiple configurations, allowing different applications to use their own credentials through a centralized configuration structure:
+
+```elixir
+# Default configuration
+config :peek_app_sdk,
+  peek_app_secret: "DEFAULT_SECRET",
+  peek_app_id: "DEFAULT_APP_ID",
+  peek_api_url: "https://api.peek.com",
```

Comment:
Sorry one more thing - I think the **default** should actually be the public gateway (especially since this repo is public). if you want to override this in the app suite to bypass the gateway, that can be private.